internal/services/newrelic/new_relic_monitor_resource.go

```diff
@@ -352,11 +389,62 @@ func (r NewRelicMonitorResource) Read() sdk.ResourceFunc {
 				state.UserInfo = flattenUserInfoModel(properties.UserInfo)
 			}
 
+			monitorId := monitoredsubscriptions.NewMonitorID(id.SubscriptionId, id.ResourceGroupName, id.MonitorName)
+			monitoredSubscriptionResp, err := metadata.Client.NewRelic.MonitoredSubscriptionsClient.Get(ctx, monitorId)
+			if err != nil && !response.WasNotFound(monitoredSubscriptionResp.HttpResponse) {
+				return fmt.Errorf("retrieving NewRelic Monitored Subscriptions of %s: %+v", *id, err)
+			}
+			if !response.WasNotFound(monitoredSubscriptionResp.HttpResponse) {
+				if monitoredSubscriptionResp.Model != nil && monitoredSubscriptionResp.Model.Properties != nil {
+					state.MonitoredSubscription = flattenMonitorSubscriptionList(monitoredSubscriptionResp.Model.Properties.MonitoredSubscriptionList)
+				}
+			}
+
 			return metadata.Encode(&state)
 		},
 	}
 }
 
+func (r NewRelicMonitorResource) Update() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 30 * time.Minute,
+		Func: func(ctx context.Context, metadata sdk.ResourceMetaData) error {
+			id, err := monitors.ParseMonitorID(metadata.ResourceData.Id())
+			if err != nil {
+				return err
+			}
+
+			var originalModel NewRelicMonitorModel
```

Comment:
I think this can just be called model?
```suggestion
			var model NewRelicMonitorModel
```

Reply:
fixed